src/assets/js/protocol/protocol-navigation.js

```diff
@@ -0,0 +1,82 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// create namespace
+if (typeof Mozilla === 'undefined') {
+    var Mozilla = {};
+}
+
+(function() {
+    'use strict';
+
+    var Navigation = {};
+    var menu;
+    var _options = {
+        onNavOpen: null
+    };
+
+    /**
+     * Event handler for navigation menu button `click` events.
+     */
+    Navigation.onClick = function(e) {
+        e.preventDefault();
+
+        // Update button state
+        e.target.classList.toggle('is-active');
+
+        // Update menu state
+        menu.classList.toggle('is-open');
+
+        // Update aria-expended state on menu.
+        var expanded = menu.classList.contains('is-open') ? true : false;
+        menu.setAttribute('aria-expanded', expanded);
+
+        if (expanded) {
+            if (typeof _options.onNavOpen === 'function') {
+                _options.onNavOpen();
+            }
+        } else {
+            if (typeof _options.onNavClose === 'function') {
+                _options.onNavClose();
+            }
+        }
+    };
+
+    /**
+     * Set initial ARIA navigation states.
+     */
+    Navigation.setAria = function() {
+        menu.setAttribute('aria-expanded', false);
+    };
+
+    /**
+     * Bind navigation event handlers.
+     */
+    Navigation.bindEvents = function() {
+        menu = document.querySelector('.mzp-c-navigation-items');
```

Comment:
Do you think we could we name this variable something other than menu? Maybe nav-items?